components/LandingPage/PageComponent.vue

```diff
@@ -99,15 +157,18 @@
 
   }
 
-  .categories-btn:after {
-    color: #fff;
-    font-weight: bold;
-    float: right;
-    margin-left: 5px;
-  }
+  /*.categories-btn:hover {*/
+  /*  background-color: #950ca0;*/
+  /*  color: #fff;*/
+  /*}*/
```

Comment:
```suggestion
  /*.categories-btn:hover {*/
  /*  background-color: #950ca0;*/
  /*  color: #fff;*/
  /*}*/
```
Δεν χρειαζονται, μπορουν να αφαιρεθουν